starlingx/inventory/v1/osds/requests.go

```diff
@@ -89,7 +89,7 @@ func (opts *OSDOpts) fixUnits() {
 
 // Create accepts a CreateOpts struct and creates a new volume group using the
 // values provided.
-func Create(c *gophercloud.ServiceClient, opts OSDOpts) (r CreateResult) {
+func Create(c *gophercloud.ServiceClient, hostid string, opts OSDOpts) (r CreateResult) {
```

Comment:
What is the reason for this change? Changes don't appear to have been made to callers of this function elsewhere

Reply:
This was reverted